configs/sites/gaea/modules.yaml

```diff
@@ -0,0 +1,8 @@
+modules:
+  default:
+    enable::
+    - tcl
+    #tcl:
+    #  projections:
+    #    git: 'Core/{name}/{version}'
+    #    git-lfs: 'Core//{name}/{version}'
```

Comment:
Not a big deal, but could you fix the newline thing?

Reply:
Fixed.